packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.m

```diff
@@ -436,8 +440,11 @@ - (int)errorCodeNativeToJS:(SKErrorCode)errorCode
     case SKErrorInvalidSignature:
     case SKErrorInvalidOfferPrice:
     case SKErrorInvalidOfferIdentifier:
+    case SKErrorOverlayInvalidConfiguration:
+    case SKErrorIneligibleForOffer:
       return 13;
     case SKErrorMissingOfferParams:
+    case SKErrorUnsupportedPlatform:
```

Comment:
Do you know what triggers this? Is it e.g. Catalyst code on macOS for an app that supports IAPs only on iOS?

Reply:
Sorry, I don't know what triggers `SKErrorUnsupportedPlatform`. The Apple docs are empty (https://developer.apple.com/documentation/storekit/skerrorcode/skerrorunsupportedplatform).

I will reassign this to 0 (UNKNOWN) for now. (I've also commented the magic numbers to show which IAPErrorCode enum value they correspond to).